code/modules/jobs/job_types/_job.dm

```diff
@@ -193,8 +192,7 @@
 			back = duffelbag //Department duffel bag
 		else
 			back = backpack //Department backpack
-	if((H.jumps == SUIT) && (!isnull(jumpsuit)))	//isnull check to ensure we don't remove the assigned uniform if jumpsuit/jumpskirt aren't set
-		uniform = jumpsuit
+
 	else if((H.jumps == SKIRT) && (!isnull(jumpskirt)))
```

Comment:
prob will fail travis

Reply:
```suggestion
	if((H.jumps == SKIRT) && (!isnull(jumpskirt)))
```

Reply:
Definitely will, give me a sec

Reply:
lol I know what it is, cobby, I don't need info like this I need info on object oriented coding,  a la how a variable's child can be taken and put into another variable, as you put in your first comment but never expanded on. If you don't know how, and I don't know how, this is as simplified as it can be.